cli/src/main.rs

```diff
@@ -81,6 +81,11 @@ struct Cli {
     #[arg(long, hide = true)]
     markdown_help: bool,
 
+    /// Set log filter value [ Off, Error, Warn, Info, Debug, Trace ]
+    #[arg(long)]
+    #[arg(default_value_t = LevelFilter::Info)]
+    log: LevelFilter,
```

Comment:
I know I wrote the issue, but I think the parameter should rather be `--log-level`.

Also could you change this so that lowercase levels work? I don't want to write `--log-level DEBUG` but rather `--log-level debug`.

Reply:
> I know I wrote the issue, but I think the parameter should rather be `--log-level`.

Yes, I update the PR for this :D

> Also could you change this so that lowercase levels work? I don't want to write `--log-level DEBUG` but rather `--log-level debug`.

The "clap" package very smart, we can --log-level DEBUG/debug/Debug, they can understand any.